app/views/organizations/new.html.erb

```diff
@@ -32,5 +32,9 @@
         </div>
       <% end %>
     <% end %>
+
+    <div class="protip">
+      <%= octicon('light-bulb') %> <strong>ProTip!</strong> Don't see your organization? <a href="https://github.com/settings/applications">Make sure you have granted us access.</a>
```

Comment:
Better yet, you can link to the exact page using this: https://developer.github.com/v3/oauth/#directing-users-to-review-their-access-for-an-application